packages/data-stores/src/reader/types/index.ts

```diff
@@ -89,12 +89,13 @@ export type PostSubscription = {
 	site_icon: string;
 	site_url: string;
 	domain: string;
+	is_wpforteams_site: boolean;
+	is_paid_subscription: boolean;
```

Comment:
There's some `paid_subscription` code being included here, OK to look past this.